src/Period.php

```diff
@@ -28,6 +28,24 @@ public static function days(int $numberOfDays): Period
         return new static($startDate, $endDate);
     }
 
+    public static function months(int $numberOfMonths): Period
+    {
+        $endDate = Carbon::today();
+
+        $startDate = Carbon::today()->subMonths($numberOfMonths)->startOfMonth();
```

Comment:
I'm thinking that `months(6)` should give the data of the entire past 6 months. We should lose the `startOfMonth` call. 

I'm thinking that `months(6)` should give the data of the entire past 6 months. We should lose the `startOfMonth` call. 

Same with the `years` function.

For really custom methods we could just make the class macroable.

Reply:
I see, I did kinda of hesitate doing this actually due usage sample I would like to share. I updated the code and Readme. Kindly check again.